services/graylog/scripts/configure.py

```diff
@@ -139,17 +145,96 @@ def configure_email_notifications(_session, _headers):
+        if (
+            len(
+                [
+                    noti
+                    for noti in _r.json()["notifications"]
+                    if noti["title"]
+                    == "Graylog " + MACHINE_FQDN + " Slack notification"
+                ]
+            )
+            == 0
+        ):
+            raw_data = (
+                '{"title":"Graylog '
+                + MACHINE_FQDN
+                + """ Slack notification","description":"Slack notification","config": {
+	    	    "color": "#FF0000",
+	    	    "webhook_url": \""""
+                + GRAYLOG_SLACK_WEBHOOK_URL
+                + """\",
+	    	    "channel": "#"""
+                + GRAYLOG_SLACK_WEBHOOK_CHANNEL
+                + """\",
+                "custom_message":"--- [Event Definition] ---------------------------\\nTitle:       ${event_definition_title}\\nType:        ${event_definition_type}\\n--- [Event] --------------------------------------\\nTimestamp:            ${event.timestamp}\\nMessage:              ${event.message}\\nSource:               ${event.source}\\nKey:                  ${event.key}\\nPriority:             ${event.priority}\\nAlert:                ${event.alert}\\nTimestamp Processing: ${event.timestamp}\\nTimerange Start:      ${event.timerange_start}\\nTimerange End:        ${event.timerange_end}\\nEvent Fields:\\n${foreach event.fields field}\\n${field.key}: ${field.value}\\n${end}\\n${if backlog}\\n--- [Backlog] ------------------------------------\\nLast messages accounting for this alert:\\n${foreach backlog message}\\n"""
+                + "https://monitoring."
+                + MACHINE_FQDN
+                + "/graylog/messages"
+                + """/${message.index}/${message.id}\\n${end}${end}\\n",
+	    	    "user_name": "Graylog",
+	    	    "notify_channel": true,
+	    	    "link_names": false,
+	    	    "icon_url": \""""
+                + GRAYLOG_SLACK_WEBHOOK_ICON_URL
+                + """\",
+	    	    "icon_emoji": "",
+	    	    "backlog_size": 5,
+	    	    "type": "slack-notification-v1"}}"""
+            )
+            _r = _session.post(
+                _url, headers=_hed, verify=False, data=raw_data.encode("utf-8")
+            )
+            if _r.status_code == 200:
+                print("Slack Notification added with success !")
+                # Keeping notification ID
+                _r = _session.get(_url, headers=_hed, verify=False)
+                _slack_notification_id = [
+                    noti
+                    for noti in _r.json()["notifications"]
+                    if noti["title"]
+                    == "Graylog " + MACHINE_FQDN + " Slack notification"
+                ][0]["id"]
+                return _slack_notification_id
+            print(
+                "Error while adding the Slack Notification. Status code of the request : "
+                + str(r.status_code)
+                + " "
+                + r.text
+            )
+            sys.exit(1)
```

Comment:
check https://docs.python.org/3/library/os.html#os.EX_OK for all these error codes

Reply:
thx, I changed it to this